src/core/authentication/authentication.service.spec.ts

```
@@ -352,7 +354,7 @@ describe('AuthenticationService', () => {
   describe('shouldExtendSession', () => {
     beforeEach(() => {
       // Mock the constructor call that sets extendSessionMinRemainingTTL
-      jest
+      vi
         .spyOn(service as any, 'parseEarliestPossibleExtend')
         .mockReturnValue(300000); // 5 minutes
```

ESLint

[error] Replace `⏎········.spyOn(service·as·any,·'parseEarliestPossibleExtend')⏎········.mockReturnValue(300000` with `.spyOn(service·as·any,·'parseEarliestPossibleExtend').mockReturnValue(⏎········300000⏎······`

(prettier/prettier)